unit_tests/unit/test_cluster.py

```
@@ -977,3 +977,61 @@ def test_base_node_subclass_constructs_with_forwarded_kwargs(cls, monkeypatch):
+    """
+    kwargs = _parent_create_node_kwargs(cls)
+    if kwargs is None:
+        pytest.skip(f"{cls.__name__} has no ancestor with `_create_node`")
+
+    try:
+        cls._create_node(cls.__new__(cls), **kwargs)
+    except TypeError as exc:
+        if any(phrase in str(exc) for phrase in ("unexpected keyword argument", "got multiple values", "missing ")):
+            raise
+    except Exception:  # noqa: BLE001
+        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)